.github/workflows/trigger-e2e.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       - name: Check if user is part of Isomer core team
         uses: tspascoal/get-user-teams-membership@v1
         id: checkUserMember
+        continue-on-error: true
```

Comment:
Hey! Am a bit hesitant to allow continue for **all** errors, is there a way to state continue on error for only `workflow_dispatch` errors?

Reply:
If there is an error, the workflow will continue processing but the `ISOMER_RUN_E2E` variable will not be set to `true`, thus causing the rest of the workflow to be skipped unless we know who triggered the workflow. Basically, the rest of the workflow will run only if the teams membership check is successful, and will not run if otherwise.